### ISSUE TITLE
Running on server causes crash at import of configurator gui

### ISSUE THREAD

**Issue by Muehli25** [sibalzer/impfbot#88]
I am running the application on a server. When running version 3.0.0 it does crash when starting.
Stacktrace:
```
File "/mnt/user/opensoftware/impfbot/src/config_generator.py", line 2, in <module>
    import tkinter as tk
  File "/usr/lib64/python3.9/tkinter/__init__.py", line 37, in <module>
    import _tkinter # If this fails your Python may not be configured for Tk
```
An easy fix is by only import the config generator and thereby tkinter, when needed.

**Comment by sibalzer** [comment]
We must look into a better fix than #89

Proposal by @paulypeter:

> Could you try wrapping the tk import in a try-except block in config-generator.py instead?
> Does it a raise a ModuleNotFoundError or an ImportError?
> 
> try:
>     import tkinter as tk
>     from tkinter import ttk
> except Exception:
>     ...


https://github.com/sibalzer/impfbot/pull/89#issuecomment-857427795

**Comment by Muehli25** [comment]
Full Stacktrace:
```
Traceback (most recent call last):                                                                                                                    
  File "/mnt/user/opensoftware/impfbot/src/impfbot.py", line 8, in <module>                                                                           
    from config_generator import start_config_generation                                                                                              
  File "/mnt/user/opensoftware/impfbot/src/config_generator.py", line 2, in <module>                                                                  
    import tkinter as tk                                                                                                                              
  File "/usr/lib64/python3.9/tkinter/__init__.py", line 37, in <module>                                                                               
    import _tkinter # If this fails your Python may not be configured for Tk                                                                          
ImportError: libtk8.6.so: cannot open shared object file: No such file or directory
```

**Comment by paulypeter** [comment]
Thank you!

**Comment by sibalzer** [comment]
closed with #98
